rust/signed_doc/src/validator/rules/mod.rs

```diff
@@ -25,6 +25,10 @@ pub(crate) use template::{ContentRule, ContentSchema};
 
 /// Struct represented a full collection of rules for all fields
 pub(crate) struct Rules {
+    /// 'id' field validation rule
+    pub(crate) id: Option<IdRule>,
```

Comment:
Maybe you've done it for now just for convenience of the development, but they should not been optional, they must always been present